packages/utilities/psammead-assets/src/svgs/plainIcons.jsx

```diff
@@ -0,0 +1,23 @@
+/* Plain Icons contain svgs which are b64 transformed and so cannot depend on any classes
```

Comment:
this comment is misleading because the b64 is up to the consumer of this package